<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="212959" started-at="2015-12-28T09:36:48Z" finished-at="2015-12-28T09:40:20Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="212959" started-at="2015-12-28T09:36:48Z" finished-at="2015-12-28T09:40:20Z">
      <class name="com.epam.seleniumTask.tests.concrete.TaskTestSuite">
        <test-method status="PASS" signature="runBrovser()[pri:0, instance:com.epam.seleniumTask.tests.concrete.TaskTestSuite@57d5872c]" name="runBrovser" is-config="true" duration-ms="2559" started-at="2015-12-28T12:36:48Z" finished-at="2015-12-28T12:36:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runBrovser -->
        <test-method status="FAIL" signature="runSuite(com.epam.seleniumTask.tests.DataLoader)[pri:0, instance:com.epam.seleniumTask.tests.concrete.TaskTestSuite@57d5872c]" name="runSuite" duration-ms="206842" started-at="2015-12-28T12:36:50Z" data-provider="testData" finished-at="2015-12-28T12:40:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.epam.seleniumTask.tests.DataLoader@35d08e6c]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.epam.seleniumTask.tests.concrete.TaskTestSuite.simpleInitialization(TaskTestSuite.java:39)
	at com.epam.seleniumTask.tests.concrete.TaskTestSuite.runSuite(TaskTestSuite.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runSuite -->
        <test-method status="FAIL" signature="runSuite(com.epam.seleniumTask.tests.DataLoader)[pri:0, instance:com.epam.seleniumTask.tests.concrete.TaskTestSuite@57d5872c]" name="runSuite" duration-ms="3192" started-at="2015-12-28T12:40:17Z" data-provider="testData" finished-at="2015-12-28T12:40:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.epam.seleniumTask.tests.DataLoader@5911e990]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.epam.seleniumTask.tests.concrete.TaskTestSuite.simpleInitialization(TaskTestSuite.java:39)
	at com.epam.seleniumTask.tests.concrete.TaskTestSuite.runSuite(TaskTestSuite.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runSuite -->
        <test-method status="PASS" signature="endWork()[pri:0, instance:com.epam.seleniumTask.tests.concrete.TaskTestSuite@57d5872c]" name="endWork" is-config="true" duration-ms="342" started-at="2015-12-28T12:40:20Z" finished-at="2015-12-28T12:40:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endWork -->
      </class> <!-- com.epam.seleniumTask.tests.concrete.TaskTestSuite -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
